scripts/copy-artifacts.js

```diff
@@ -48,9 +49,13 @@ copyDappToolsArtifact(dappToolsArtifacts, 'WormholeJoin', output)
 copyDappToolsArtifact(dappToolsArtifacts, 'WormholeConstantFee', output)
 copyDappToolsArtifact(dappToolsArtifacts, 'WormholeOracleAuth', output)
 copyDappToolsArtifact(dappToolsArtifacts, 'WormholeRouter', output)
-copyHardhatArtifact(join(hardhatArtifacts, 'l1/L1DAITokenBridge.sol/L1DAITokenBridge.json'), output)
-copyHardhatArtifact(join(hardhatArtifacts, 'l2/dai.sol/Dai.json'), output)
-copyHardhatArtifact(join(hardhatArtifacts, 'l2/L2DAITokenBridge.sol/L2DAITokenBridge.json'), output)
-copyHardhatArtifact(join(hardhatArtifacts, 'l1/L1DAIWormholeBridge.sol/L1DAIWormholeBridge.json'), output)
-copyHardhatArtifact(join(hardhatArtifacts, 'l1/L1Escrow.sol/L1Escrow.json'), output)
-copyHardhatArtifact(join(hardhatArtifacts, 'l2/L2DAIWormholeBridge.sol/L2DAIWormholeBridge.json'), output)
+copyHardhatArtifact(join(hardhatBridgeArtifacts, 'l1/L1DAITokenBridge.sol/L1DAITokenBridge.json'), output)
+copyHardhatArtifact(join(hardhatBridgeArtifacts, 'l2/dai.sol/Dai.json'), output)
+copyHardhatArtifact(join(hardhatBridgeArtifacts, 'l2/L2DAITokenBridge.sol/L2DAITokenBridge.json'), output)
+copyHardhatArtifact(join(hardhatBridgeArtifacts, 'l1/L1DAIWormholeBridge.sol/L1DAIWormholeBridge.json'), output)
+copyHardhatArtifact(join(hardhatBridgeArtifacts, 'l1/L1Escrow.sol/L1Escrow.json'), output)
+copyHardhatArtifact(join(hardhatBridgeArtifacts, 'l2/L2DAIWormholeBridge.sol/L2DAIWormholeBridge.json'), output)
+copyHardhatArtifact(join(hardhatTestArtifacts, 'TestBadDebtPushSpell.sol/TestBadDebtPushSpell.json'), output)
+copyHardhatArtifact(join(hardhatTestArtifacts, 'TestBadDebtPushSpell.sol/DaiJoinLike.json'), output)
+copyHardhatArtifact(join(hardhatTestArtifacts, 'TestBadDebtPushSpell.sol/VatLike.json'), output)
```

Comment:
do you need these additional artifacts?

Reply:
They are not needed for the tests to pass but `yarn typecheck` fails without them.